### PR TITLE
[TS] Added a typescript example for the knight token

### PR DESCRIPTION
**Pull request by junkil-park**
### Description
* added the typescript example for the knight/food token
* added the metadata and images for the token

### Test Plan
`pnpm knight`

### CODE REVIEW

**Review by movekevin — COMMENTED** [review]
Looks great! As discussed offline, can you write a simple walkthrough tutorial for this?

**Review by movekevin — APPROVED** [review]

File: ecosystem/typescript/sdk/examples/typescript/knight.ts [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  await faucetClient.fundAccount(admin.address(), 100_000_000);
+  await faucetClient.fundAccount(user.address(), 100_000_000);
+  console.log(
+    "\nCompile and publish the knight module (`aptos-core/aptos-move/move-examples/token_objects/knight`) using the following profile, and press enter:",
+  );
+  console.log(` knight_admin:`);
+  console.log(` private_key: "${admin.toPrivateKeyObject().privateKeyHex}"`);
+  console.log(` public_key: "${admin.pubKey()}"`);
+  console.log(` account: ${admin.address()}`);
+  console.log(` rest_url: "https://fullnode.devnet.aptoslabs.com"`);
```
Comment (0xmaayan) [inline ecosystem/typescript/sdk/examples/typescript/knight.ts:151]: nit
```suggestion
  console.log(` rest_url: ${NODE_URL}`);
  console.log(` faucet_url: ${FAUCET_URL}`);
```

**Review by 0xmaayan — APPROVED** [review]

File: ecosystem/typescript/sdk/examples/typescript/knight.ts (hunk 1 above)
Comment (junkil-park) [reply ecosystem/typescript/sdk/examples/typescript/knight.ts:151]: Thanks! Done!

File: aptos-move/move-examples/token_objects/knight/sources/food.move [hunk 2]
```diff
@@ -81,21 +81,21 @@ module token_objects::food {
             sender,
             string::utf8(b"Corn Token Description"),
             string::utf8(CORN_TOKEN_NAME),
-            string::utf8(b"https://raw.githubusercontent.com/junkil-park/metadata/main/knight/Corn"),
```
[the remark this answers is not in the document]
Comment (junkil-park) [reply aptos-move/move-examples/token_objects/knight/sources/food.move:84]: Done. Thanks!

**Pull request merged**